package.json

```diff
@@ -49,22 +51,16 @@
     "@types/cookie-parser": "^1.4.7",
     "@types/cors": "^2.8.17",
     "@types/express": "^4.17.21",
+    "@types/jest": "^29.5.12",
     "@types/jsonwebtoken": "^9.0.6",
     "@types/multer": "^1.4.11",
-    "@types/node": "^20.12.7",
+    "@types/node": "^20.12.5",
+    "@types/supertest": "^6.0.2",
     "@types/swagger-ui-express": "^4.1.6",
-    "@types/validator": "^13.11.9",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "eslint": "^8.57.0",
-    "eslint-config-standard-with-typescript": "^43.0.1",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-n": "^16.6.2",
-    "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-react": "^7.34.1",
-    "globals": "^15.0.0",
     "husky": "^9.0.11",
-    "pg": "^8.11.5",
-    "sequelize-cli": "^6.6.2",
```

Comment:
Why are you removing this?

Reply:
@SergeInGithub You mean I should create a new branch and leave this one??

Reply:
> Why are you removing this?

I installed it locally since it was causing conflicts and pushed changes.

Reply:
i think these are your teammates work, I'm asking why you're deleting these dependencies

Reply:
> i think these are your teammates work, I'm asking why you're deleting these dependencies

Actually I didn't delete it, here is a screenshot for more information on how this dependency is my dev-dependency and this changes are my latest push:
![Husky-package](https://github.com/atlp-rwanda/e-commerce-serge-30-bn/assets/77304295/e4f1f065-3e94-48da-9c17-47f10c26031a)